From efa905c6b20db4486e9ee6ab991ecfcd7a04ce63 Mon Sep 17 00:00:00 2001
From: Phil Sutter <psutter@redhat.com>
Date: Fri, 11 Mar 2016 15:38:15 +0100
Subject: [PATCH] whitespace cleanup

Bugzilla: https://bugzilla.redhat.com/show_bug.cgi?id=1315930
Upstream Status: iproute2.git commit 3d0b7439dfac9
Conflicts: Just picking changes to tc/m_simple.c

commit 3d0b7439dfac96d93386295a4c961b89bd3d8463
Author: Stephen Hemminger <shemming@brocade.com>
Date:   Sat Dec 20 15:47:17 2014 -0800

    whitespace cleanup

    Remove all trailing whitespace and space before tabs.
---
 tc/m_simple.c | 38 +++++++++++++++++++-------------------
 1 file changed, 19 insertions(+), 19 deletions(-)

diff --git a/tc/m_simple.c b/tc/m_simple.c
index b61aca7..866552f 100644
--- a/tc/m_simple.c
+++ b/tc/m_simple.c
@@ -9,7 +9,7 @@
  * Authors:	J Hadi Salim <jhs@mojatatu.com>
  *
  * Pedagogical example. Adds a string that will be printed every time
- * the simple instance is hit. 
+ * the simple instance is hit.
  * Use this as a skeleton action and keep modifying it to meet your needs.
  * Look at linux/tc_act/tc_defact.h for the different components ids and
  * definitions used in  this actions
@@ -26,20 +26,20 @@
  *
   hadi@noma1:$ tc qdisc add dev eth0 ingress
   hadi@noma1:$tc filter add dev eth0 parent ffff: protocol ip prio 5 \
-   	 u32 match ip protocol 1 0xff flowid 1:1 action simple "Incoming ICMP"
-  
+	 u32 match ip protocol 1 0xff flowid 1:1 action simple "Incoming ICMP"
+
   hadi@noma1:$ sudo tc -s filter ls  dev eth0 parent ffff:
-   filter protocol ip pref 5 u32 
-   filter protocol ip pref 5 u32 fh 800: ht divisor 1 
-   filter protocol ip pref 5 u32 fh 800::800 order 2048 key ht 800 bkt 0 flowid 1:1 
+   filter protocol ip pref 5 u32
+   filter protocol ip pref 5 u32 fh 800: ht divisor 1
+   filter protocol ip pref 5 u32 fh 800::800 order 2048 key ht 800 bkt 0 flowid 1:1
      match 00010000/00ff0000 at 8
-     	action order 1: Simple <Incoming ICMP>
-     	 index 4 ref 1 bind 1 installed 29 sec used 29 sec
-     	 Action statistics:
-     		Sent 0 bytes 0 pkt (dropped 0, overlimits 0 requeues 0) 
-     	 	backlog 0b 0p requeues 0 
-  
-  
+	action order 1: Simple <Incoming ICMP>
+	 index 4 ref 1 bind 1 installed 29 sec used 29 sec
+	 Action statistics:
+		Sent 0 bytes 0 pkt (dropped 0, overlimits 0 requeues 0)
+		backlog 0b 0p requeues 0
+
+
   hadi@noma1$ ping -c 1 www.google.ca
   PING www.google.ca (74.125.225.120) 56(84) bytes of data.
   64 bytes from ord08s08-in-f24.1e100.net (74.125.225.120): icmp_req=1 ttl=53 time=31.3 ms
@@ -52,15 +52,15 @@
   [135354.473951] simple: Incoming ICMP_1
 
   hadi@noma1$ sudo tc/tc -s filter ls  dev eth0 parent ffff:
-  filter protocol ip pref 5 u32 
-  filter protocol ip pref 5 u32 fh 800: ht divisor 1 
-  filter protocol ip pref 5 u32 fh 800::800 order 2048 key ht 800 bkt 0 flowid 1:1 
+  filter protocol ip pref 5 u32
+  filter protocol ip pref 5 u32 fh 800: ht divisor 1
+  filter protocol ip pref 5 u32 fh 800::800 order 2048 key ht 800 bkt 0 flowid 1:1
     match 00010000/00ff0000 at 8
 	action order 1: Simple <Incoming ICMP>
 	 index 4 ref 1 bind 1 installed 206 sec used 67 sec
 	Action statistics:
-	Sent 84 bytes 1 pkt (dropped 0, overlimits 0 requeues 0) 
-	backlog 0b 0p requeues 0 
+	Sent 84 bytes 1 pkt (dropped 0, overlimits 0 requeues 0)
+	backlog 0b 0p requeues 0
 */
 
 #include <stdio.h>
@@ -82,7 +82,7 @@
 static void explain(void)
 {
 	fprintf(stderr, "Usage: ... simple STRING\n"
-		"STRING being an arbitrary string\n" 
+		"STRING being an arbitrary string\n"
 		"example: \"simple blah\"\n");
 }
 
-- 
1.8.3.1

